.github/workflows/release-automation.yaml

```diff
@@ -80,17 +77,18 @@ jobs:
         with:
           fetch-depth: 0
           ref: main
-          token: ${{ secrets.HEDERA_BOT_TOKEN }}
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          persist-credentials: 'true'
 
       - name: Import GPG Key
         id: gpg_importer
-        uses: crazy-max/ghaction-import-gpg@01dd5d3ca463c7f10f7f4f7b4f177225ac661ee4 # v6.1.0
+        uses: step-security/ghaction-import-gpg@6c8fe4d0126a59d57c21f87c9ae5dd3451fa3cca # v6.1.0
```

Comment:
Does the comment version `# v6.1.0` need to be updated with the change of commit id?

Reply:
it does not. step-security is now maintaining their initial version with a version matching the source action.

See the step security released version here: https://github.com/step-security/ghaction-import-gpg/releases/tag/v6.1.0

The commit id associated with that version matches what's in this action.